src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
+        List<Excavator> excavators = new ArrayList<>();
+        excavators.add(new Excavator("orange", "Komatsu",
+                "construction", "rock"));
+        excavators.add(new Excavator("orange", "Volvo",
+                "piping", "suction-pipe"));
```

Comment:
I could be wrong because my work hasn't been reviewed yet. But it seems to me that it is more correct to initialize and fill the list in the constructor.